nvdaHelper/client/sconscript

```diff
@@ -30,9 +30,13 @@ controllerRPCHeader,controllerRPCClientSource=env.MSRPCStubs(
 
 clientLibName="nvdaControllerClient%s"%('64' if env['TARGET_ARCH']=='x86_64' else '32')
 
+rcFile=env.Command('nvda.rc',env['projectRCFile'],Copy('$TARGET','$SOURCE'))
+RESFile=env.RES(rcFile)

```

Comment:
Is there a reason we can't use the same .res file for all of these dlls/exes, rather than having a separate one for each.